books/RayTracingTheNextWeek.html

```diff
@@ -739,11 +750,9 @@
     [Listing [sphere-bbox]: <kbd>[sphere.h]</kbd> Sphere with bounding box]
 </div>
 
-<div class='together'>
-For `moving sphere`, we want the bounds of all places the moving sphere could occupy while it moves.
-To do this, we can take the box of the sphere at time = 0, and the box of the sphere at time = 1,
-and compute the box of those two boxes. We'll add a new `aabb` constructor that takes two boxes as
-input below.
+For `moving sphere`, we want the bounds of its entire range of motion. To do this, we can take the
+box of the sphere at time=0, and the box of the sphere at time=1, and compute the box of those two
+boxes.
```

Comment:
"compute the box of these two boxes"
->
"compute the box that contains both of these two boxes"
?